odoo/addons/base/populate/res_user.py

```diff
@@ -30,13 +32,19 @@ def generate_partner_id(iterator, *args):
         def get_company_ids(values, **kwargs):
             return [(6, 0, [values['company_id']])]
 
+        def compute_login(values=None, counter=0, **kwargs):
+            return f'user_login_{last_id + counter + 1}'
+
+        def compute_name(values=None, counter=0, **kwargs):
+            return f'user_login_{last_id + counter + 1}'
+
         return [
             ('active', populate.cartesian([True, False], [0.9, 0.1])),
             ('partner_id', generate_partner_id),
             ('company_id', populate.randomize(self.env.registry.populated_models['res.company'])),
             ('company_ids', populate.compute(get_company_ids)),
-            ('login', populate.constant('user_login_{counter}')),
-            ('name', populate.constant('user_{counter}')),
```

Comment:
Note: just making populate of base models re-entrant to significantly improve its usage and debugging